blink/README.md

```diff
@@ -0,0 +1,34 @@
+# BlinkOps Integration
+
+## Overview
+
+The Datadog integration for [Blink][1] enables you to trigger automations in response to new incidents. You can also create, update, and view incidents and events in the Datadog event stream.
```

Comment:
```suggestion
## Overview

The [Blink][1] integration enables you to:
1. Trigger automations within Blink in response to new Datadog incidents 
2. Create and update Datadog incidents in Blink 
3. View any created incidents or events in the Datadog event stream
```

Reply:
Included some information on the integration but feel free to edit if I misunderstood.

Reply:
Information looks good, committed the change